ui-tests/src/main/java/io/jenkins/plugins/analysis/warnings/AbstractDetailsTable.java

```diff
@@ -10,42 +10,31 @@
 
 import org.jenkinsci.test.acceptance.po.PageObject;
 
-/**
- * Class representing an issues-table on the {@link AnalysisResult}.
- *
- * @author Stephan Plöderl
- */
-public class IssuesTable {
+public abstract class AbstractDetailsTable {
```

Comment:
Bitte den JavaDoc gerade ziehen (am besten das Checkstyle Plugin installieren in IntelliJ).